### PR TITLE
updates to vignette

### PR DESCRIPTION
**Pull request by mhines-usgs**

### CODE REVIEW

File: vignettes/hazardItems.Rmd [hunk 1]
```diff
@@ -34,6 +35,22 @@ This package **hazardItems** provides services in support of coastal change haza
...
+#Installation
+To install **hazardItems** 
+
+Install required packages:
+```{r, eval=FALSE}
+install.packages(c("httr","scales","rjson","png","maps","mapdata","XML","devtools","roxygen2","knitr","rmarkdown","mapdata"))
+```
+
+Then:
+```{r, eval=FALSE}
```
Comment (jread-usgs) [inline vignettes/hazardItems.Rmd]: At this point, I don't this `hazardItems` is actually installed, just all of its dependencies. but...it isn't up on GRAN (owi.usgs.gov/R) yet, so we can leave those instructions as a to do.

File: vignettes/hazardItems.Rmd [hunk 2]
```diff
@@ -86,7 +103,7 @@ An example workflow for adding or updating a real time storm would involve the f
 
 An example workflow for a *new* incoming storm would involve the following:
 
-* a new `template` item is added from the mediation page (publish/item) describing the incoming storm (manually performed by NACCH administrators)
+* a new `template` item is added from the <a href="http://marine.usgs.gov/coastalchangehazardsportal/publish/item/" target="_blank">mediation page</a> describing the incoming storm (manually performed by NACCH administrators)
```
Comment (jread-usgs) [inline vignettes/hazardItems.Rmd]: coooool

**Comment by mhines-usgs** [comment]
will that comment regarding dependencies make the 'install required packages' part irrelevant?

**Comment by jread-usgs** [comment]
yes, as long as the package is hosted on GRAN, but in the meantime, the dependency support isn't there. GRAN is fun :+1:

**Pull request merged**